vortex-dtype/src/serde/mod.rs

```diff
@@ -30,4 +30,20 @@ mod test {
             }],
         );
     }
+
+    #[test]
+    fn test_serde_dtype() {
+        assert_tokens(&DType::from(PType::U8), &[Token::Struct {
+            name: "DType",
+            len: 2,
+        }]);
```

Comment:
interesting, not familiar with this. what are tokens in this context?

Reply:
from the serde crate (turns out the original version of this test was a bug / incomplete, womp)